### PR TITLE
Update Nautobot Config Environment

### PR DESCRIPTION
**Pull request by jvanderaa**
### Fixes: #353 

- Uses a default of `"6379"` instead of `6379` in the os.getenv() statement for REDIS_PORT
- Changes `REDIS_SSL` to use `os.getenv()` instead of `os.environ.get()` for consistency with the rest of the environment variables loading.

### CODE REVIEW

File: nautobot/core/tests/nautobot_config.py [hunk 1]
```diff
@@ -45,12 +45,12 @@ def is_truthy(arg):
 
 # Redis variables
 REDIS_HOST = os.getenv("REDIS_HOST", "localhost")
-REDIS_PORT = os.getenv("REDIS_PORT", 6379)
+REDIS_PORT = os.getenv("REDIS_PORT", "6379")
```
Comment (glennmatthews) [inline nautobot/core/tests/nautobot_config.py:48]: Just checking, does this need to be cast to `int` for safety?

Comment (jathanism) [reply nautobot/core/tests/nautobot_config.py:48]: This isn't necessary. Django handles it internally.

File: nautobot/core/templates/nautobot_config.py.j2 [hunk 2]
```diff
@@ -23,7 +23,7 @@ DATABASES = {
         "PASSWORD": os.getenv("NAUTOBOT_PASSWORD", ""),  # Datbase password
         "HOST": os.getenv("NAUTOBOT_DB_HOST", "localhost"),  # Database server
         "PORT": os.getenv("NAUTOBOT_DB_PORT", ""),  # Database port (leave blank for default)
-        "CONN_MAX_AGE": os.getenv("NAUTOBOT_DB_TIMEOUT", 300),  # Database timeout
+        "CONN_MAX_AGE": int(os.getenv("NAUTOBOT_DB_TIMEOUT", "300")),  # Database timeout
```
Comment (jathanism) [inline nautobot/core/templates/nautobot_config.py.j2:26]: This isn't necessary either. Django handles the string/int conversions internally.

File: nautobot/core/tests/nautobot_config.py [hunk 3]
```diff
@@ -45,12 +45,12 @@ def is_truthy(arg):
...
 REDIS_PASSWORD = os.getenv("REDIS_PASSWORD", "")
 
 # Check for Redis SSL
 REDIS_SCHEME = "redis"
-REDIS_SSL = is_truthy(os.environ.get("REDIS_SSL", False))
+REDIS_SSL = is_truthy(os.getenv("REDIS_SSL", "False"))
```
Comment (jathanism) [inline nautobot/core/tests/nautobot_config.py:53]: no need to make `False` into a string. I like the `os.getenv` change though for consistency.

**Review by jathanism — CHANGES_REQUESTED** [review]
Please just remove all the changes except for `os.getenv`.

**Comment by jathanism** [comment]
Please disregard my initial review. I think it would be good to shore up all of the integers to strings and the `is_truthy` argument as you had.

Please give the same treatment to:

- `development/nautobot_config.py`
- `nautobot/core/tests/nautobot_config.py`
- `nautobot/core/settings.py`
- `nautobot/core/templates/nautobot_config.py.j2`

And make sure to catch the port numbers and `MAX_CONN_AGE` for `DATABASES`! :)

**Pull request closed without merging**